### PR TITLE
Added all files and README

### PR DESCRIPTION
**Pull request by theAdityaNVS**
## Pull Request Checklist

Go through the check boxes given below and make sure you mark the relevant ones.
#### To check put x inside [ ] eg. [x] Extension

Is your project an
- [✔️ ] Extension 
- [ ] Application

Given below are the mandatory requirements we expect from your project. Your Pull Request will be reviewed only if all the minimum requirements relevant to your project are satisfied.

- [✔️ ] Added readme.md to my project folder with relevant information.
- [✔️ ]  I have provided the demo video link (showing the functionality in action) or the public deployment link of my extension of application within the readme.md (if Applicable).
- [✔️ ] I have explained the process in a readable manner within readme.md.

### Terms and Conditions

`Plagiarism check from Team Crio will be solid and if your code is found to be plagiarized, the team has every right to reject your Pull Request and take necessary actions.`
- [✔️ ] I can assure you that this is my own contribution and I did not involve in any kind of plagiarism activities.

Buckle up and wait till you receive communication from the review team. Keep a tab on this pull request page for comments on your project. Any necessary suggestions will also be provide here. All the best :)

### CODE REVIEW

**Comment by sudhanshutiwari264** [comment]
@theAdityaNVS you've done a great job :clinking_glasses: it matches the structure I had planned :+1: 
@archithdwij could you please review this!
Let's wait until he reviews! 
Celebrate! Take a moment to relax!

**Review by sudhanshutiwari264 — APPROVED** [review]

File: Meet Up - JavaScript/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# chrome-extension-meetup
+Its a simple extension which can open and store all the added meeting links to join quickly.
```
Comment (Nitesh-thapliyal) [inline Meet Up - JavaScript/README.md]: There are some grammatical mistakes present, please correct them.

**Comment by Nitesh-thapliyal** [comment]
Good work @theAdityaNVS 

Here is a suggestion that you can work on :

- Buttons are overlapping the Links, can you make that right?

Screenshot (1158)

**Comment by Nitesh-thapliyal** [comment]
Hey Aditya, 

I just found a bug in your extension:

- Rename feature is not working instead it is creating a new link and extra tabs.
For better understanding I have created a video, go through the video and try to fix it asap
 


https://user-images.githubusercontent.com/53345517/124399139-2d6d2700-dd37-11eb-8d85-5bc909c6c6c1.mp4

**Review by Nitesh-thapliyal — CHANGES_REQUESTED** [review]
Hey Aditya,

I just found a bug in your extension:

- Rename feature is not working instead it is creating a new link with extra tabs.

For better understanding I have created a video, go through the video and try to fix it asap

https://user-images.githubusercontent.com/53345517/124399330-432f1c00-dd38-11eb-9f24-ccaa7f76ad8f.mp4

**Comment by theAdityaNVS** [comment]
Hi,
- So the empty list error is fixed now.
- It also seems like Heroku isn't allowing few functions so I deployed it on GitHub Pages [Here](https://theadityanvs.github.io/chrome-extension-meetup/).
- Also I have fixed padding issues as bootstrap CDN broke and i fixed it.
- I have also updated README and added screenshot.

File: Meet Up - JavaScript/README.md (hunk 1 above)
Comment (theAdityaNVS) [reply Meet Up - JavaScript/README.md]: Yes I have updated the README

**Review by Nitesh-thapliyal — APPROVED** [review]
LGTM

File: Meet Up - JavaScript/README.md [hunk 2]
```diff
@@ -1,15 +1,29 @@
-# chrome-extension-meetup
-Its a simple extension which can open and store all the added meeting links to join quickly.
+# Chrome-Extension-MeetUp
+Its a simple extension which store all your meeting links together in one place and in one click you can join quickly.
```
Comment (Nitesh-thapliyal) [inline Meet Up - JavaScript/README.md:2]: Still, there is grammatical mistake use  Grammarly to avoid grammatical mistakes

**Review by Nitesh-thapliyal — APPROVED** [review]

**Review by Nitesh-thapliyal — APPROVED** [review]

**Comment by archithdwij** [comment]
Hi @theAdityaNVS I like the idea of managing the meeting links in one place. Your work looks solid :) 
I have a small suggestion and that's because I am just curious. 
Is it possible to link the meetings scheduled in the calendar/Gmail and update the extension automatically? Just a thought. Do it as a future upgrade or something if it's possible. Other than that, congratulations on getting your PR merged 💯

**Review by jainaayush01 — CHANGES_REQUESTED** [review]
Please remove the ".DS_Store" files from the Pull Request. 
Just add it to .gitignore, the file has no relevance on other computers.

**Review by archithdwij — APPROVED** [review]

**Comment by theAdityaNVS** [comment]
Yes @archithdwij  , Will definitely try to implement and yes thats possible through calendar integration. 
But to implement in extension it's a bit complicated due to chrome content security policies. 

Also the PR didn't get merged.

**Comment by theAdityaNVS** [comment]
@jainaayush01 
I have removed those files.

**Comment by archithdwij** [comment]
I saw aayush's suggestions after my review so I waited for you to remove the .DS_Store file. Will get it merged now :)

**Review by archithdwij — APPROVED** [review]

**Comment by archithdwij** [comment]
@all-contributors Please add @theAdityaNVS for code, design, idea

**Pull request merged**